docs/protocol/cli.md

```diff
@@ -278,36 +278,35 @@ Ubuntu:
 
 ```bash
 
-./subspace-cli-ubuntu-x86_64-v3-v0.1.12 init
+./subspace-cli-ubuntu-x86_64-v3-v0.3.2-alpha init
 
 ```
 Ubuntu Executable (aarch64):
 
 ```bash
 
-./subspace-cli-ubuntu-aarch64-v0.1.12 init
+./subspace-cli-ubuntu-aarch64-v0.3.2-alpha init
 
 ```
 
 </TabItem>
 </Tabs>
 
-This will prompt you to setup your CLI configurations to begin farming. You should see a similar prompt like so:
+This will prompt you to setup your CLI configurations to begin farming. You should see a similar prompt like so (some info might be different than shown here w.r.t your OS):
 
 ```bash
-version: 0.1.0
-
-SUBSPACE NETWORK
+version: 0.3.2
 
 Configuration creation process has started...
 Enter your farmer/reward address: REDACTED_ADDRESS
-Enter your node name to be identified on the network(defaults to `username`, press enter to use the default):
-Specify a plot location (press enter to use the default: `"/home/username/.local/share/subspace-cli/plots"`):
+Enter your node name to be identified on the network(defaults to `ozgunozerk`, press enter to use the default):
```

Comment:
This section is a bit odd with your username on it, might make sense to keep it as `username` but just my opinion.

Reply:
ah yeah wonderful catch!